src/components/help/help.pw.tsx

```diff
@@ -0,0 +1,354 @@
+  });
+
+  (["error", "add", "minus", "settings"] as any[]).forEach((iconType) => {
+    test(`should render with iconType prop passed as ${iconType}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<HelpComponentTest isFocused type={iconType} />);
+
+      await expect(icon(page)).toBeVisible();
+      await expect(icon(page)).toHaveAttribute("type", iconType);
+    });
+  });
+});
+
+test.describe("Accessibility tests for Help component", () => {
+  testData.forEach((ariaLabel) => {
+    test(`should check ariaLabel as ${ariaLabel}`, async ({ mount, page }) => {
+      await mount(<HelpComponentTest ariaLabel={ariaLabel} />);
+      await checkAccessibility(page);
+    });
+  });
+
+  testData.forEach((className) => {
+    test(`should check className as ${className}`, async ({ mount, page }) => {
+      await mount(<HelpComponentTest className={className} />);
+      await checkAccessibility(page);
+    });
+  });
+
+  testData.forEach((children) => {
+    test(`should check children as ${children}`, async ({ mount, page }) => {
+      await mount(<Help> {children} </Help>);
+      await checkAccessibility(page);
+    });
+  });
+
+  testData.forEach((helpId) => {
+    test(`should check id as ${helpId}`, async ({ mount, page }) => {
+      await mount(<HelpComponentTest helpId={helpId} />);
+      await checkAccessibility(page);
+    });
+  });
+
+  ["https://carbon.sage.com", "google", "https://www.google.com"].forEach(
+    (hrefLink) => {
+      test(`should check href link as ${hrefLink}`, async ({ mount, page }) => {
+        await checkAccessibility(page);
+      });
+    }
+  );
+
+  [true, false].forEach((boolVal) => {
+    test(`should check when isFocused prop is passed as ${boolVal}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(
+        <HelpComponentTest isFocused={boolVal}>{tooltipText}</HelpComponentTest>
+      );
+      await checkAccessibility(page);
+    });
+  });
+
+  colors.forEach(([names, color]) => {
+    test(`should check tooltipBgColor as ${names}`, async ({ mount, page }) => {
+      await mount(
+        <HelpComponentTest tooltipBgColor={color} isFocused>
+          {tooltipText}
+        </HelpComponentTest>
+      );
+      await checkAccessibility(page, "color-contrast");
+    });
+  });
+
+  colors.forEach(([names, color]) => {
+    test(`should check tooltipFontColor as ${names}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(
+        <HelpComponentTest tooltipFontColor={color} isFocused>
+          {tooltipText}
+        </HelpComponentTest>
+      );
+      await checkAccessibility(page, "color-contrast");
+    });
+  });
+
+  (["top", "bottom", "left", "right"] as HelpProps["position"][]).forEach(
+    (position) => {
+      test(`should render tooltipFlipOverrides as ${position}`, async ({
+        mount,
+        page,
+      }) => {
+        await mount(
+          <HelpComponentTest tooltipFlipOverrides={[position]} isFocused>
```

Comment:
The tooltip position here is not flipping between the positions. It is always rendered in the top position. You need to apply the same viewport, Box and left margin restrictions here that you did in the main prop test case (line 166-185) to force the tooltip position to flip.